src/time.cr

```diff
@@ -252,6 +252,13 @@ struct Time
   # :nodoc:
   DAYS_PER_4_YEARS = 365*4 + 1
 
+  # This constant is defined to be "1970-01-01 00:00:00 UTC" on all systems with respect to the system clock.
+  # Can be used to create a `Time::Span` that represents an unix epoch time duration.
+  # ```
+  # Time.utc_now - Time::UNIX_EPOCH
+  # ```
+  UNIX_EPOCH = Time.utc 1970, 1, 1
+
   # :nodoc:
   UNIX_SECONDS = SECONDS_PER_DAY.to_i64 * (1969*365 + 1969/4 - 1969/100 + 1969/400)
```

Comment:
if `UNIX_SECONDS` is no longer used and has a `:nodoc:` then we should remove it (or schedule to remove it)

Reply:
I was planning to use exclusively `Time::UNIX_EPOCH.total_seconds`, but I've encounter issues; possibility chicken and egg ones, because `Time::UNIX_EPOCH` is a `Time` inside a `Time` object.
To sum-up, `UNIX_SECONDS` is still in use, for now.